core/src/main/res/layout/nav_main.xml

```diff
@@ -17,18 +16,14 @@
   ~
   -->
 
-<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
   android:layout_width="match_parent"
   android:layout_height="match_parent">
 
-  <androidx.fragment.app.FragmentContainerView
-    android:id="@+id/fragment_custom_app_container"
+  <ImageView
     android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    app:layout_constraintBottom_toBottomOf="parent"
-    app:layout_constraintEnd_toEndOf="parent"
-    app:layout_constraintStart_toStartOf="parent"
-    app:layout_constraintTop_toTopOf="parent" />
-
-</androidx.constraintlayout.widget.ConstraintLayout>
+    android:layout_height="wrap_content"
```

Comment:
removing marginTop and using `android:fitsSystemWindows="true"` would be better. 
Using margin may vary outputs on different devices. 
Here's an example :
![Webp net-resizeimage (5)](https://user-images.githubusercontent.com/53831987/89781039-2d1cd980-db30-11ea-8714-f503cc1f5e00.png)
`android:fitsSystemWindows="true"` will work fine here.:smile:

Reply:
I suspected something would be odd with the margin. Thanks!